src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -293,6 +299,17 @@ public static VariantContext swapRefAlt(final VariantContext vc, final Collectio
         return swappedBuilder.make();
     }
 
+    /**
+     * Checks whether the reference allele in the provided variant context actually matches the reference sequence
+     *
+     * @param alleles list of alleles from which to find the reference allele
+     * @param referenceSequence the ref sequence
+     * @return true if they match, false otherwise
+     */
+    protected static boolean referenceAlleleMatchesReference(final List<Allele> alleles, final String referenceSequence) {
+        final Allele refAllele = alleles.stream().filter(Allele::isReference).findAny().orElse(null);
```

Comment:
shouldn't there always be a reference allele?

Reply:
Yes, you're right.  I'll fix.